<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="testcases.LoginFunctionalityWithDataProvider">
        <methods>
          <include name="setup"/>
          <include name="tearDown"/>
          <include name="validatingLoginWithDifferentUserData"/>
        </methods>
      </class> <!-- testcases.LoginFunctionalityWithDataProvider -->
      <class name="testcases.LoginFunctionalityWithScreenshotInExtentReport">
        <methods>
          <include name="setup"/>
          <include name="validateLoginFunctionalityWithScreenshotInExtentReport"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- testcases.LoginFunctionalityWithScreenshotInExtentReport -->
      <class name="testcases.LoginFunctionalityWithExtentReport">
        <methods>
          <include name="validateLoginFunctionality"/>
          <include name="tearDown"/>
          <include name="setup"/>
        </methods>
      </class> <!-- testcases.LoginFunctionalityWithExtentReport -->
      <class name="testcases.loginFunctionality">
        <methods>
          <include name="setup"/>
          <include name="tearDown"/>
          <include name="validateValidLogin"/>
        </methods>
      </class> <!-- testcases.loginFunctionality -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
